### DISCUSSION TITLE
dns lookup for local names question

### DISCUSSION THREAD

**Discussion by zadigre** [AdguardTeam/AdGuardHome#3050] [Q&A]
hello everyone!

I have ADH 0.160.1 installed. 
ADH is my DHCP and DNS server on my home network. 

if I do a reverse dns lookup on lets say 192.168.25.1, I get the name of my computer which is "laptop"
If I do a dns look-up for "laptop", I get NXDOMAIN. but if I do "laptop.lan", I get the IP address.

I was under the impression that nslookup laptop alone would work.
Is it supposed to work like that or I'm missing something?

**Comment by ameshkov** [comment]
@ainar-g please take a look

**Comment by ainar-g** [comment]
Hello and sorry for the slow response.  DHCP hosts are only available using the local domain name.  This is working as intended.  You could also add a `$dnsrewrite` rule or an `/etc/hosts` entry on the server to get `nslookup laptop` to work.
